pkg/agent/openflow/network_policy.go

```diff
@@ -2066,6 +2066,9 @@ type featureNetworkPolicy struct {
 	// egressTables map records all IDs of tables related to egress rules.
 	egressTables map[uint8]struct{}
 
+	loggingGroupCache sync.Map
```

Comment:
better to add a comment what's the key and value.

Reply:
updated.